src/Microsoft.IdentityModel.Tokens/opensource/json/DefaultJsonNameTable.cs

```diff
@@ -53,12 +55,13 @@ public DefaultJsonNameTable()
         }
 
         /// <summary>
-        /// Gets the string containing the same characters as the specified range of characters in the given array.
+        /// Gets a string containing the same characters as the specified range of characters in the given array.
         /// </summary>
         /// <param name="key">The character array containing the name to find.</param>
         /// <param name="start">The zero-based index into the array specifying the first character of the name.</param>
         /// <param name="length">The number of characters in the name.</param>
-        public override string Get(char[] key, int start, int length)
+        /// <returns>A string containing the same characters as the specified range of characters in the given array.</returns>
+        public override string? Get(char[] key, int start, int length)
```

Comment:
>string? [](http://example.com/codeflow?start=24&length=7)

huh, TIL about nullable reference types... [https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-reference-types](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-reference-types)

Reply:
That is the code in the 13.0.2 branch.